### PR TITLE
extract knockout/reviveSwitch into switchHelper

### PR DESCRIPTION
**Pull request by andriidovhan**
This PR the same as  #3244 
I found to hard rebase mentioned PR on the develop branch, so I've just created the new PR.

The main idea of this pr is add knockout/reviveSwitch methods into switchHelper
Also two specs are improved:
- remove needless `timeLoop` in `AutoRerouteV2Spec`
-  add missing annotation in `FlowPingSpec`

### CODE REVIEW

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy [hunk 1]
```diff
@@ -332,4 +355,62 @@ class SwitchHelper {
             assert Math.abs(expected - actual) <= 1
         }
     }
+
+    /**
+     * Disconnect a switch from controller either removing controller settings inside an OVS switch
+     * or blocking access to floodlight via iptables for a hardware switch.
+     *
+     * @param sw - switch which is going to be disconnected.
+     * @param waitForRelatedLinks - it is an optional parameter. Please, use this parameter if you want to be sure
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy]: I feel a bit uncomfortable with 'please' and 'you' words in docs. Make it more neutral and laconic

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy [hunk 2]
```diff
@@ -102,7 +102,7 @@ class AutoRerouteSpec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
-    @Tags([VIRTUAL, LOW_PRIORITY])
+//    @Tags([VIRTUAL, LOW_PRIORITY])
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy]: ?

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy (hunk 1 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy]: fixed

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy (hunk 2 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy]: my bad
fixed

**Pull request merged**